doc/lua_api.md

```diff
@@ -4142,6 +4142,11 @@ Helper functions
     * returns time with microsecond precision. May not return wall time.
 * `table.copy(table)`: returns a table
     * returns a deep copy of `table`
+    * strips metatables, but this may change in the future
```

Comment:
IMO we don't need the metatable-related behavior to change in the future now that `table.copy_with_metatable()` is a separate function.

Reply:
don't need it to, but i think this would be a sane(r) default; it's just for backwards compat that we aren't changing the behavior of `table.copy` right away.

when would you want `table.copy` to strip metatables? i think that's the niche("r") use case.